.github/workflows/orchestrator.yml

```
@@ -0,0 +1,47 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  orchestrator-lint-report:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install tox
+        run: pip install tox
+      - name: Run tests using tox
+        run: |
+          charms=$(find ./orchestrator-bundle/ -name tox.ini)
```

```suggestion
          set -eux
          charms="$(find ./orchestrator-bundle/ -name tox.ini)"
```

Worth bailing early on errors in CI with `set -eux` and the `-x` will help to debug any issues. I'd recommend quoting variables/command expansions too.

---

(Applies elsewhere)

---

Fixed